pom.xml

```diff
@@ -34,7 +34,7 @@
         <hibernate-types-52.version>2.10.3</hibernate-types-52.version>
         <spring-security-core.version>5.4.5</spring-security-core.version>
         <cql2pgjson.version>32.2.0</cql2pgjson.version>
-        <log4j2.version>2.16.0</log4j2.version> <!-- if less then vulnerable to CVE-2021-44228 and CVE-2021-45046-->
+        <log4j2.version>2.17.2</log4j2.version>
```

Comment:
This commit is only partly included:
https://github.com/folio-org/mod-data-export-spring/pull/173/commits/8c3b73d548308c57491668db637191352d9da9f9